src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -18,19 +21,38 @@ import TableHeader from "components/molecules/TableHeader";
 import { UserList } from "components/molecules/UserList";
 import { TablesUserList } from "components/molecules/TablesUserList";
 
+import { BackButton } from "components/atoms/BackButton";
+
 import { TABLES } from "./constants";
 
 import "./ConversationSpace.scss";
 
+export type ConversationSpaceProps = {
+  venue?: WithId<AnyVenue>;
+};
+
 // @debt refactor this to pass in venue as a prop
-export const ConversationSpace: React.FunctionComponent = () => {
-  const venue = useSelector(currentVenueSelectorData);
+export const ConversationSpace: React.FunctionComponent<ConversationSpaceProps> = ({
+  venue,
+}) => {
+  const { parentVenue } = useRelatedVenues({
+    currentVenueId: venue?.id,
+  });
+  const parentVenueId = parentVenue?.id;
```

Comment:
[NB] We should just add another export in `useRelatedVenues` to access `parentVenueId` directly.

Reply:
I think this is oos for this pr, but I can make note of it and make a new pr when I get a chance

Reply:
Not worth OOS'ing it, it's like a 2 line change. Please do it here.

Reply:
fixed in 2d0f35d4bf277771e395c265f64f28e959340a74